README.md

```diff
@@ -44,94 +98,211 @@ Open inbound ports on your security group. Use the table below as a guide. Make
+   _Boardwalk_ has functionality to export metadata to Broad's
+   FireCloud. In order to use it you need to provide your Google Cloud
+   Platform credentials.
+
+   You will also be asked for a Google site verfication code. For this
+   field you can just enter `NONE`.
+
+1. The bagit-firecloud-lambda is use for the **Export to Firecloud**
+   button.
+
+1. The [S3 bucket you created earlier](#makebucket) is what you should
+   use for this next step.
+
+1. The dos-dss server is needed next. Again you can use a prexisting one
+   if this is just for development.
+
+### <a name="prod"></a> Installing in `prod` mode
 
-### Running the Installer
-Once you have collected the above information, clone the repository on VM and run the bootstrap script. Be sure to set your branch to `feature/commons` as these instruction are specific to this branch.
+Once the above steps have been completed we are now ready to install the
+components of the CGP.  In `prod` mode the installation will run the
+Docker containers for all of the components listed below from the
+respective images from *Quay.io*. The `nginx` docker will be built from
+the *nginx-image* directory.
 
-    $ git clone https://github.com/DataBiosphere/cgp-deployment.git
-    $ cd cgp-deployment
-    $ sudo bash install_bootstrap
+Note that this decision can be made for _Common_ and _Boardwalk_
+independently. For details regarding _Boardwalk_ see the
+[README](boardwalk/README.md).
 
-The `install_bootstrap` script will ask you to configure each service interactively. Specifically, you need to decide on whether you require a production (`prod` mode) or a development (`dev` mode) environment. Note that this decision can be made for _Common_ and _Boardwalk_ independently. For details regarding _Boardwalk_ see the [README](boardwalk/README.md).
 
-#### Installing in `prod` mode
-Once the above steps have been completed we are now ready to install the components of the CGP.
-In `prod` mode the installation will run the Docker containers for all of the components listed below from the respective images from *Quay.io*. The `nginx` docker will be built from the *nginx-image* directory.
+### <a name="prod"></a> Installing in `dev` mode
 
+Setting up *Common* (the gateway nginx server) to run in `dev` mode will
+cause [Let's Encrypt](https://letsencrypt.org/) to issue fake SSL
+certificates, which won't exhaust your certificate's limit. Setting up
+*Boardwalk* to run in `dev` mode will first build then run the Docker
+containers `boardwalk_nginx`, `boardwalk_dcc-dashboard`,
+`boardwalk_dcc-dashboard-service`, and `boardwalk_boardwalk` from the
+images (see
+[here](https://github.com/DataBiosphere/cgp-deployment/blob/feature/update-readme/boardwalk/README.md#development-mode)
+for more details). In addition, the `nginx` image is built from the
+*nginx-dev* directory. If your work requires real SSL certificates
+during development, it is recommended to set up *Common* in `prod` mode,
```

Comment:
Maybe put this at the very beginning of the section, i.e., even if running in dev mode, we recommend running Common in prod mode. Otherwise it's easy (for me) to overlook, and I think you ran into this too when you set up your first instance.